tracer/src/Datadog.Trace/Agent/MessagePack/SpanMessagePackFormatter.cs

```diff
@@ -412,7 +413,8 @@ private int WriteTags(ref byte[] bytes, int offset, in SpanModel model, ITagProc
             offset += MessagePackBinary.WriteStringBytes(ref bytes, offset, _languageValueBytes);
 
             // add "version" tags to all spans whose service name is the default service name
-            if (string.Equals(span.Context.ServiceNameInternal, model.TraceChunk.DefaultServiceName, StringComparison.OrdinalIgnoreCase))
+            var serviceNameEqualsDefault = string.Equals(span.Context.ServiceNameInternal, model.TraceChunk.DefaultServiceName, StringComparison.OrdinalIgnoreCase);
```

Comment:
Is "ignore case" correct here? 🤔

Reply:
Specifically the `StringComparison.OrdinalIgnoreCase`?

Yes

```
Requirements about _dd.base_service:

comparison between the current value of the service tag and the value of the service name (e.g. from $DD_SERVICE) is case insensitive;

The value of _dd.base_service can be set without any normalization, as the agent will apply the same normalization as applied to the service name.

```